utils/logger.js

```diff
@@ -19,8 +19,22 @@ const flatstr = require('flatstr');
 const newConsole = require('console').Console;
 const config = require('../config');
 
-const output = fs.createWriteStream(config.log.file, { flags: 'a' });
-const myConsole = new newConsole(output, output);
+const defaultOutput = '/dev/stdout';
+
+if (!Array.isArray(config.log.output)) {
+	if (typeof config.log.output === 'string') {
+		config.log.output = [config.log.output];
+	} else if (typeof config.log.file === 'string') {
```

Comment:
I don't really like this config file "backward compatibility". ;)

Reply:
The backward compatibility is not the main reason for that. Potential users might still want to use a single output the most intuitive way possible.